diff --git a/src/App.css b/src/App.css
index e3c9d10..a3c6ec7 100644
--- a/src/App.css
+++ b/src/App.css
@@ -1,3 +1,11 @@
+.app-container{
+  height: 100vh;
+}
+
+.navbar {
+  margin-bottom: 50px;
+}
+
 .btnheader .nav-link {
   border: none;
   border-radius: 10px;
@@ -17,3 +25,5 @@
   background-color: #ea7c5a;
   color: white !important;
 }
+
+
diff --git a/src/App.jsx b/src/App.jsx
index e4ff416..bc8b87e 100644
--- a/src/App.jsx
+++ b/src/App.jsx
@@ -1,16 +1,17 @@
 import React, { useState, useEffect } from "react";
 import Footer from "./components/footer";
-import Card from "./components/card";
+// import Card from "./components/card";
 import Cards from "./components/cards"
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom"
 import Header from "./components/header";
-import Login from "./components/Login";
+// import Login from "./components/Login";
 import Carousel from "./components/carousel"
 
-import Modulet from "./components/Modulet"
+// import Modulet from "./components/Modulet"
 
 import SuperAdmin from "./components/SuperAdmin";
 import Lesson from "./components/lesson"
+import "./App.css"
 
 
 
@@ -40,23 +41,22 @@ const App = () => {
   }, []);
 
   return (
-    <div>
+    <div className="app-container">
 
       <Router>
         <Header/>
-        <Carousel/>
         <Routes>
-          <Route path='/' element={<Cards/>}>  </Route>
+          <Route path='/' element={<Carousel/>}>  </Route>
+          <Route path='/home-lessons' element={<Cards/>}>  </Route>
           <Route path='/admin' element={<SuperAdmin/>}>  </Route>
           <Route path='/foro' element={""}>  </Route>
           <Route path='/lesson' element={<Lesson/>}> </Route>
         </Routes>
 
-        <Modulet/>
+        {/* <Modulet/> */}
         <Footer/>
       </Router>
 
-      
     </div>
   );
 };
diff --git a/src/assets/styles/Login.css b/src/assets/styles/Login.css
index 73b0459..80ae1bd 100644
--- a/src/assets/styles/Login.css
+++ b/src/assets/styles/Login.css
@@ -1,10 +1,10 @@
 
 *{
-    
+
 }
 .wrapper{
     background: #fff;
-    padding: 50px 20px 0 20px;
+    padding: 50px 20px 50px 20px;
 }
 
 .main{
@@ -61,7 +61,6 @@
 .input-box{
     width: 330px;
     box-sizing: border-box;
-    
 }
 
 .input-box header{
@@ -70,6 +69,9 @@
     margin-bottom: 45px;
 
 }
+.input-box{
+    padding-left: 30px;
+}
 
 .input-field{
     display:flex;
diff --git a/src/components/cards.js b/src/components/cards.js
index 6af3a90..042e283 100644
--- a/src/components/cards.js
+++ b/src/components/cards.js
@@ -8,20 +8,20 @@ import img3 from '../assets/img3.png'
 const cards = [
     {
         id: 1,
-        title: "Leccion 1",
+        title: "Lección 1",
         image: img1,
         text: 'Esta es la primera lección, trata de:',
         url: 'http://localhost:3000/lesson'
     },
     {
         id: 2,
-        title: "Leccion 2",
+        title: "Lección 2",
         image: img2,
         url: 'http://localhost:3000/lesson'
     },
     {
         id: 3,
-        title: "Leccion 3",
+        title: "Lección 3",
         image: img3,
         url: 'http://localhost:3000/lesson'
     }
diff --git a/src/components/carousel.js b/src/components/carousel.js
index 4f64f33..be91bb9 100644
--- a/src/components/carousel.js
+++ b/src/components/carousel.js
@@ -1,13 +1,15 @@
 import React from 'react'
+import { Link } from "react-router-dom";
+import '../assets/styles/carousel.css'
 
 import banner1 from '../assets/banner1.png'
 import banner2 from '../assets/banner2.png'
 import banner3 from '../assets/banner3.png'
 import banner4 from '../assets/banner4.png'
 
-function Carousel() {
+const carousel = () => {
   return (
-    <div>
+    <div className='carousel-container'>
         <div id="carouselExampleIndicators" class="carousel slide" data-bs-ride="carousel">
             <div className="carousel-indicators">
                 <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
@@ -26,7 +28,7 @@ function Carousel() {
                     <img src={banner3} className="d-block w-100 m-0" alt="..."/>
                 </div>
                 <div className="carousel-item">
-                    <img src={banner4} className="d-block w-100 m-0" alt="..."/>
+                    <Link to="/home-lessons"><img src={banner4} className="d-block w-100 m-0" alt="..." /></Link>
                 </div>
             </div>
             <button className="carousel-control-prev" type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide="prev">
@@ -43,4 +45,4 @@ function Carousel() {
   )
 }
 
-export default Carousel
\ No newline at end of file
+export default carousel
\ No newline at end of file
diff --git a/src/components/footer.css b/src/components/footer.css
index edf3f86..17b4bf0 100644
--- a/src/components/footer.css
+++ b/src/components/footer.css
@@ -1,3 +1,11 @@
+.footer-container{
+    margin-top: 50px;
+    position:relative;
+    left:0;
+    bottom:0;
+    right:0;
+}
+
 .orange-text{
     color: #F73F1D;
 }
diff --git a/src/components/footer.js b/src/components/footer.js
index 5930a4a..1401f72 100644
--- a/src/components/footer.js
+++ b/src/components/footer.js
@@ -8,7 +8,7 @@ import './footer.css'
 const Footer =()=>{
 
     return(
-        <div>
+        <div className="footer-container">
             <footer className='text-black py-4 bg-light border-top p-3'>
                 <div className='container'>
                     <nav className='row'>
@@ -18,7 +18,6 @@ const Footer =()=>{
                         <ul className='col-12 col-md-3 list-unstyled'>
                             <li className='orange-text font-weight-bold mb-4 text-center'>Dirección</li>
                             <li className='text-center'>Tv. 73d Bis A #68 36 <br></br> SUR, Bogotá</li>
-                            
                         </ul>
                         <ul className='col-12 col-md-3 list-unstyled'>
                             <li className='orange-text font-weight-bold mb-4 text-center'>Contactos</li>
